components/tools/OmeroWeb/omeroweb/webclient/controller/container.py

```diff
@@ -1257,11 +1258,19 @@ def remove(self, parents, index, tag_owner_id=None):
                         self.conn.deleteObjectDirect(al._obj)
                 # if comment is orphan, delete it directly
                 orphan = True
+
+                # Use delete Dry Run...
+                cid = self.comment.getId()
+                command = Delete2(targetObjects={"CommentAnnotation": [cid]},
+                                  dryRun=True)
+                handle = self.conn.c.sf.submit(command)
```

Comment:
NB: `sf.submit() && conn._waitOnCmd()` can be shortened to `cb = self.conn.c.submit()`